### PR TITLE
Add Institut Pasteur config

### PR DESCRIPTION
**Pull request by rplanel**
Add nf-core config for the Institut Pasteur cluster using Slurm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ Currently documentation is available for the following clusters:
 * [UPPMAX-DEVEL](docs/uppmax-devel.md)
 * [UPPMAX](docs/uppmax.md)
 * [UZH](docs/uzh.md)
+* [PASTEUR](docs/pasteur.md)
```
Comment (drpatelh) [inline README.md]: Thanks @rplanel . Would you mind adding this in alphabetical order please :smile:

Comment (apeltzer) [reply README.md]: My bad, will fix this once home 😆

File: nfcore_custom.config [hunk 2]
```diff
@@ -34,6 +34,7 @@ profiles {
   uzh          { includeConfig "${params.custom_config_base}/conf/uzh.config" }
   prince       { includeConfig "${params.custom_config_base}/conf/prince.config" }
   bigpurple    { includeConfig "${params.custom_config_base}/conf/bigpurple.config" }
+  pasteur      { includeConfig "${params.custom_config_base}/conf/pasteur.config" }
```
Comment (drpatelh) [inline nfcore_custom.config]: Would be great if you can do the same here too. It seems to have lost its alphabeticity somehow!

Comment (apeltzer) [reply nfcore_custom.config]: I guess we weren't really enforcing it at the moment, so thats why (or I wasn't enforcing it 😆  when reviewing but agree that it helps with general readability...)

File: docs/pasteur.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+module load Python/3.6.0
+module load java
+module load singularity
+cd /path/to/nf-core/workflows
+virtualenv .venv -p python3
+. .venv/bin/activate
+```
+2. Install nf-core: [here](https://nf-co.re/tools#installation)
+3. Get nf-core pipeline and container: [here](https://nf-co.re/tools#downloading-pipelines-for-offline-use)
+4. Get the nf-core Pasteur profile: [here](https://github.com/nf-core/rnaseq/blob/master/docs/usage.md#--custom_config_base)
```
Comment (drpatelh) [inline docs/pasteur.md]:
```suggestion
4. Get the nf-core Pasteur profile: [here](https://github.com/nf-core/configs#offline-usage)
```

**Comment by drpatelh** [comment]
Ah, youve merged already!

**Comment by apeltzer** [comment]
Yeah sorry, didn't check the sorting stuff and rest looked already very good...

**Comment by rplanel** [comment]
Thank you, you did all the work ! I'll take care of the nf-core.re changes.
This is a great project BTW !

**Pull request merged**